samples/custom_file/enclave.json

```
@@ -0,0 +1,14 @@
+{
+ "exe": "hellofile",
```

```suggestion
 "exe": "custom_file",
```
If you build with `ego-go build`, the executable will be named like the folder.